components/net/http_loader.rs

```diff
@@ -1293,6 +1306,10 @@ fn http_network_fetch(
                     _ => vec![],
                 };
                 *body = ResponseBody::Done(completed_body);
+                timing_ptr2
+                    .lock()
+                    .unwrap()
+                    .set_attribute(ResourceAttribute::ResponseEnd);
```

Comment:
quick q for here & block below: should `responseEnd` be set after the _send_ instead?

Reply:
No. The send operation is used to notify other parts of the code about the connection state; we want to record the current instant as the end of the response.